.github/workflows/sync_base.yaml

```diff
@@ -18,13 +18,14 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
+        id: setup-python
         with:
           python-version: '3.10'
       - uses: actions/cache@v3
         with:
           path: |
             ~/.commodore-venv/
-          key: ${{ runner.os }}-commodore-${{ hashFiles('requirements.txt') }}
+          key: ${{ runner.os }}-commodore-${{ steps.setup-python.outputs.python-version }}-${{ hashFiles('requirements.txt') }}
       - name: Install Commodore
```

Comment:
This is a good idea, but seems unrelated

Reply:
It's unrelated, except that the tests were breaking without it.

Reply:
Makes sense

Reply:
factored out into #25